systemd-swap

```diff
@@ -70,7 +70,18 @@ get_fs_type(){
   elif [[ -d $(dirname "$1") ]]; then path=$(dirname "$1");
   else ERRO "swapfc_path is invalid"
   fi
-  df "$path" --output=fstype | tail -n 1
+
+  fstype=$(df "$path" --output=fstype | tail -n 1)
+
+  if [ "$fstype" == "-" ]; then
+    if btrfs subvolume show "$path" &>/dev/null; then
+      fstype="btrfs"
+    else
+      ERRO '$swapfc_path is located on an unknown filesystem'
```

Comment:
Use `""` instead of `''` to follow style

Reply:
Maybe the intention here is to print `$swapfc_path` literally, without resolving the variable. You can do that by escaping the `$` though, i.e. `"\$swapfc_path"`. But if you want to tell the user that this variable is not set correctly, you may omit the `$` altogether, like we do for `swapfc_frequency` for example.